Northstar.CustomServers/mod/scripts/vscripts/mp/_gamestate_mp.nut

```diff
@@ -1017,3 +1018,18 @@ void function DialoguePlayWinnerDetermined()
 		PlayFactionDialogueToTeam( "scoring_lost", losingTeam )
 	}
 }
+
+void function OnPlayerChangedTeam( entity player )
```

Comment:
Maybe a short comment describing the purpose of the function would be nice?
I know we haven't done much stuff in terms of doc comments in the past but we should kinda start making this a habit to improve code readability and I guess we need to start somewhere ^^"
Can also skip if you wanna but would be nice. Up to you ^^


```suggestion
/// Does XYZ
void function OnPlayerChangedTeam( entity player )
```